qui/devices/device_widget.py

```diff
@@ -428,6 +452,28 @@ def update_single_feature(self, _vm, _event, feature, value=None, oldvalue=None)
                     self.parent_ports_to_hide.append(dev.port)
                     self.hide_child_devices(dev.port, False)
 
+    def update_resolution(self, vm, _event, _feature, value=None, _oldvalue=None):
```

Comment:
`feature`, `value` and `oldvalue` needs to be named exactly this way, they are used as keyword arguments.